apps/month.py

```diff
@@ -0,0 +1,186 @@
+# SPDX-License-Identifier: MIT
+# Copyright (C) 2023 Eloi Torrents
+"""Month
+~~~~~~~~~~~~
+
+Simple calendar that displays months and highlights the current day.
+
+Controls:
+    - Swipe up to show next month.
+    - Swipe down to show previous month.
+    - Press the button to return to current month.
+
+Icon courtesy of coffandro: https://github.com/coffandro
+
```

Comment:
The screenshot should be included here in the docstring

```suggestion

.. figure:: res/screenshots/MonthApp.png
    :width: 179
```